h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -847,11 +902,81 @@ private GLMModel runGLMModel(boolean standardize, Family family, Link link, Key<
     GLMModel model = new GLM(tempParams).trainModel().get();
     return model;
   }
+  
+  // copy from scoring_history back to _sc or _lsc
```

Comment:
comment indicates that the method is poorly named

Reply:
I renamed the function and the variables.